tests/storage/export.go

```diff
@@ -1492,6 +1495,277 @@ var _ = SIGDescribe("Export", func() {
 			Expect(postCertParamms).ToNot(Equal(preCertParamms))
 		})
 	})
+
+	var _ = Describe("virtctl vmexport command", func() {
+		const (
+			commandName    = "vmexport"
+			defaultOutput  = "/tmp/test-disk.img"
+			defaultVMEName = "vme-test"
+		)
+
+		sc := ""
+		vmeName := ""
+
+		checkForReadyExport := func(name string) {
+			vmexport, err := virtClient.VirtualMachineExport(util.NamespaceTestDefault).Get(context.Background(), name, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			waitForReadyExport(vmexport)
+		}
+
+		BeforeEach(func() {
+			storageClass, exists := libstorage.GetRWOFileSystemStorageClass()
+			if !exists {
+				Skip("Skip test when Filesystem storage is not present")
+			}
+			sc = storageClass
+			vmeName = defaultVMEName
```

Comment:
If this tests might run in parallel maybe its better to create a random name for vme and also maybe for the output file if one test might delete the just created one or something like that